otter/grade/containers.py

```diff
@@ -218,12 +223,18 @@ def kill_container():
             scores = dill.load(f)
 
         scores_dict = scores.to_dict()
-        scores_dict["percent_correct"] = scores.total / scores.possible
+        
+        scores_dict["percent_correct"] = round(scores.total / scores.possible,4)
+        scores_dict["total_points_earned"] = scores.total
 
+        pts_poss_dict = {t: [scores_dict[t]["possible"]] if type(scores_dict[t]) == dict else scores_dict[t] for t in scores_dict}
         scores_dict = {t: [scores_dict[t]["score"]] if type(scores_dict[t]) == dict else scores_dict[t] for t in scores_dict}
+        pts_poss_dict["file"] = POINTS_POSSIBLE_LABEL
+        pts_poss_dict["percent_correct"] = "--"
```

Comment:
This will interfere with libraries like pandas automatically parsing the dtype of this column as numeric. Can you set this to 0 or `NA` (which pandas reads as NaN) instead?